.github/merge_rules.yaml

```diff
@@ -250,13 +250,16 @@
 - name: XPU ATen
   patterns:
   - aten/src/ATen/xpu/**
+  - aten/src/ATen/native/mkldnn/xpu/**
   - c10/xpu/**
   - torch/csrc/xpu/**
   - torch/xpu/**
   - test/xpu/**
   - test/test_xpu.py
   - third_party/xpu.txt
   - .ci/docker/ci_commit_pins/triton-xpu.txt
+  - cmake/Modules/FindSYCLToolkit.cmake
+  - cmake/public/xpu.cmake
```

Comment:
I would be somewhat against that, as CMakeLists have an unfortunate tendency to affect global settings, even when not in use.

Reply:
Okay. I will remove the cmake-related changes.